public/app/features/alerting/unified/components/rule-editor/alert-rule-form/AlertRuleForm.tsx

```diff
@@ -290,6 +303,17 @@ export const AlertRuleForm = ({ existing, prefill }: Props) => {
       <AppChromeUpdate actions={actionButtons} />
       <form onSubmit={(e) => e.preventDefault()} className={styles.form}>
         <div className={styles.contentOuter}>
+          {existing && prefill && (
```

Comment:
I'm wondering if there's a better way to denote that a manual restore is in progress – can we work with a separate query param instead of inferring it implicitly?

Reply:
Changed to an explicit `isManualRestore` for showing this alert, have left the rest of the logic as is (existing/prefill etc)